src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -588,3 +590,14 @@ export const selectDonationAmountFiat = createSelector(
     return bnOrZero(sellAmountFiat).times(affiliatePercentage).toFixed()
   },
 )
+
+export const selectIntermediaryTransactionOutputs = createDeepEqualOutputSelector(
+  (state: SwapperState) => state.activeSwapperWithMetadata?.quote?.intermediaryTransactionOutputs,
+  (state: SwapperState) => state.trade?.intermediaryTransactionOutputs,
+  (
+    quoteIntermediaryTransactionOutputs,
+    tradeintermediaryTransactionOutputs,
+  ): IntermediaryTransactionOutput[] | undefined =>
+    // Use the trade amount if we have it, otherwise use the quote amount
+    tradeintermediaryTransactionOutputs ?? quoteIntermediaryTransactionOutputs,
```

Comment:
```suggestion
    tradeIntermediaryTransactionOutputs,
  ): IntermediaryTransactionOutput[] | undefined =>
    // Use the trade amount if we have it, otherwise use the quote amount
    tradeIntermediaryTransactionOutputs ?? quoteIntermediaryTransactionOutputs,
```